spec/support/store.rb

```diff
@@ -60,10 +60,11 @@ def self.zrange(zset_key, from, to)
 
     def self.exists(key)
       Sidekiq.redis do |r|
-        if r.respond_to?(:exists?)
-          r.exists?(key)
+        case r.exists(key)
+        when true, 1
+          true
         else
-          !!r.exists(key)
+          false
         end
```

Comment:
1 and true evaluate to a truthy value, so doing `r.exists(key) ? true : false` would do the trick here

Reply:
@juan-apa Yeah, but `0` also will evaluate to a truthy value
```ruby
irb(main):001:0> true ? 'A' : 'B'
=> "A"
irb(main):002:0> 1 ? 'A' : 'B'
=> "A"
irb(main):003:0> 0 ? 'A' : 'B'
=> "A"
irb(main):004:0> false ? 'A' : 'B'
=> "B"
```

Reply:
oh wow. I mean, makes sense, but at the same time it doesn't. It's fine by me if you keep it with the case then.